tests/lammps/test_base.py

```diff
@@ -678,3 +678,35 @@ def test_units(self):
         def setter(x):
             self.job.units = x
         self.assertRaises(ValueError, setter, "nonsense")
+
+    def test_bonds_input(self):
+        potential = pd.DataFrame({'Name': ['Morse'],
+                                  'Filename': [[]],
+                                  'Model'   : ['Morse'],
+                                  'Species' : [['Al']],
+                                  'Config'  : [['atom_style bond\n',
+                                                'bond_style morse\n',
+                                                'bond_coeff 1 0.1 1.5 2.0\n',
+                                                'bond_coeff 2 0.1 1.5 2.0']]})
+        cell = Atoms(elements=4*['Al'], positions=[[0., 0., 0.],
+                                                   [0., 2., 2.],
+                                                   [2., 0., 2.],
+                                                   [2., 2., 0.]],
+                     cell=4*np.eye(3))
+        self.job.structure = cell.repeat(2)
+        self.job.structure.bonds = [[1, 2, 1], [1, 3, 2]]
+        self.job.potential = potential
+        self.job.calc_static()
+        file_directory = os.path.join(
+            self.execution_path, "..", "static", "lammps_test_files"
+        )
+        self.job.restart_file_list.append(
+            os.path.join(file_directory, "dump.out")
+        )
+        self.job.restart_file_list.append(
+            os.path.join(file_directory, "log.lammps")
+        )
+        self.job.run(run_mode="manual")
+        self.job.status.collect = True
+        self.job.run()
```

Comment:
Again, not sure if output parsing is necessary here. After `self.job.run(run_mode="manual")`, you could inspect the `structure.inp` file to check if the bonds are written the way you really want them to be.

Reply:
Cool. I will get it fixed right away!